src/storage/page_buffer.c

```diff
@@ -14958,6 +14988,25 @@ pgbuf_bcb_is_dirty (const PGBUF_BCB * bcb)
   return (bcb->flags & PGBUF_BCB_DIRTY_FLAG) != 0;
 }
 
+static bool
+pgbuf_bcb_is_dirty_and_needs_flushing (PGBUF_BCB * bcb)
```

Comment:
inline?
`pgbuf_bcb_is_dirty` is inlined

Reply:
done